### ISSUE TITLE
plans for beta api support?

### ISSUE THREAD

**Issue by mipearson** [SumoLogic/sumologic-terraform-provider#3]
I'm going to guess "when they're not a beta anymore", but I figured I'd ask whether you plan to support beta APIs (eg, user management) within the provider

**Comment by frankreno** [comment]
That is correct, we will add them as soon as they are GA to avoid any confusion for users who are not in the Beta...

**Comment by mtanda** [comment]
@frankreno Do you have a plan to support Roles/Users management API?
https://api.sumologic.com/docs/#tag/roleManagement
https://api.sumologic.com/docs/#tag/userManagement

**Comment by frankreno** [comment]
@mtanda : we do indeed, however I do not have a date of when we will add this functionality right now but will be adding that in a future release.  We would welcome a PR if you wish to contribute the functionality as well

**Comment by mtanda** [comment]
I'm not familiar with Terraform provider, and doesn't have Sumo Logic sandbox account.
So, I wait for next release :-)

**Comment by captn3m0** [comment]
Created a separate issue for User/Role APIs: https://github.com/SumoLogic/sumologic-terraform-provider/issues/41
